app/models/study_json_record/worker.rb

```diff
@@ -48,7 +47,7 @@ def float(str)
 
   def self.reset
     StudyRelationship.study_models.each(&:delete_all)
-    StudyJsonRecord.where(version: '2').update_all(saved_study_at: nil) # rubocop:disable Rails/SkipsModelValidations
+    StudyJsonRecord.version_2.update_all(saved_study_at: nil) # rubocop:disable Rails/SkipsModelValidations
   end
```

Comment:
This was a helper method to help clean the database while testing everything, it might be better to do in a different way now that we are fully migrated, let's remove this method since we don't really need it anymore. I think it would be good to leave it, if it was more general, but it is too specific to version 2.